schemas/quiz.ts

```diff
@@ -0,0 +1,20 @@
+import { Quizoot } from "./question";
+
+export interface Quiz {
```

Comment:
It's a pre-version. Just for a first validation of the structure.

Reply:
I believe that storing the questions objects themselves is not a good idea. We should store a list of their id instead.

Reply:
You're right 😉